docs/src/app/components/pages/components/lists.jsx

```diff
@@ -39,6 +39,11 @@ export default class ListsPage extends React.Component {
 
   constructor(props) {
     super(props);
+    this.state = { selectedIndex: 1 }
+  }
+
+  handleUpdateSelectedIndex = (index) => {
+    this.setState(Object.assign({}, this.state, {selectedIndex: index}));
```

Comment:
The way I proposed to set the state ensures that whatever is currently in the state does not get overwritten / thrown away unintentionally, isn't it?